doc/source/raster_class.md

```diff
@@ -0,0 +1,337 @@
+np.sqrt(raster)
+```
+
+Logical comparison functions will cast to a {class}`~geoutils.Mask`.
+
+```{code-cell} ipython3
+# Is the raster close to another within tolerance?
+
+np.isclose(raster, raster+0.05, atol=0.1)
+```
+
+See {ref}`core-array-funcs` for more details.
+
+## Reproject
+
+Reprojecting a {class}`~geoutils.Raster` is done through the {func}`~geoutils.Raster.reproject` function, which enforces new {attr}`~geoutils.Raster.transform`
+and/or
+{class}`~geoutils.Raster.crs`.
+
+```{important}
+As with all geospatial handling methods, the {func}`~geoutils.Raster.reproject` function can be passed only a {class}`~geoutils.Raster` or {class}`~geoutils.
+Vector` as argument.
+
+A {class}`~geoutils.Raster` reference will enforce to match its {attr}`~geoutils.Raster.transform` and {class}`~geoutils.Raster.crs`.
+A {class}`~geoutils.Vector` reference will enforce to match its {attr}`~geoutils.Vector.bounds` and {class}`~geoutils.Vector.crs`.
+
+See {ref}`core-match-ref` for more details.
+```
+
+The {func}`~geoutils.Raster.reproject` function can also be passed any individual arguments such as `dst_bounds`, to enforce specific georeferencing
+attributes. For more details, see the {ref}`specific section and function descriptions in the API<api-geo-handle>`.
+
+```{code-cell} ipython3
+# Original bounds and resolution
+print(raster.res)
+print(raster.bounds)
+```
+
+```{code-cell} ipython3
+# Reproject to smaller bounds and higher resolution
+raster_reproj = raster.reproject(
+    dst_res=0.1,
+    dst_bounds={"left": 0, "bottom": 0, "right": 0.75, "top": 0.75},
+    resampling="cubic")
+raster_reproj
+```
+
+```{code-cell} ipython3
+# New bounds and resolution
+print(raster.res)
+print(raster.bounds)
+```
+
+```{note}
+In GeoUtils, `"bilinear"` is the default resampling method. A simple {class}`str` matching the naming of a {class}`rasterio.enums.Resampling` method can be
+passed.
+
+Resampling methods are listed in **[the dedicated section of Rasterio's API](https://rasterio.readthedocs.io/en/latest/api/rasterio.enums.html#rasterio.enums.Resampling)**.
+```
+
+## Crop
+
+Cropping a {class}`~geoutils.Raster` is done through the {func}`~geoutils.Raster.crop` function, which enforces new {attr}`~geoutils.Raster.bounds`.
+
+```{important}
+As with all geospatial handling methods, the {func}`~geoutils.Raster.crop` function can be passed only a {class}`~geoutils.Raster` or {class}`~geoutils.Vector`
+as argument.
+
+See {ref}`core-match-ref` for more details.
+```
+
+The {func}`~geoutils.Raster.crop` function can also be passed a {class}`list` or {class}`tuple` of bounds (`xmin`, `ymin`, `xmax`, `ymax`). By default,
+{func}`~geoutils.Raster.crop` is done in-place.
+For more details, see the {ref}`specific section and function descriptions in the API<api-geo-handle>`.
+
+```{code-cell} ipython3
+# Crop raster to smaller bounds
+raster_crop = raster.crop(crop_geom=(0.3, 0.3, 1, 1), inplace=False)
+raster_crop
+```
+
+## Polygonize
```

Comment:
For Polygonize and proximity, it may be useful to show a plot of the output, so that it's a bit more concrete.

Reply:
Maybe adding a mini-gallery... I refer to your main comment.